src/en.json

```diff
@@ -204,6 +204,251 @@
         "text": "Save your password to a secure location. If you lose your password, you will permanently lose access to your funds.",
         "title": "Password backup"
     },
+    "browser": {
+        "exceptions": {
+            "abi": {
+                "notSupport": {
+                    "message": "This version ABI not support",
+                    "hint": "Maybe, we will support this version later"
+                }
+            },
+            "default": {
+                "hints": {
+                    "sendReport": "Please, send a report about this problem"
+                },
+                "messages": {
+                    "somethingWentWrong": "Something went wrong",
+                    "failedSendToDeBot": "Failed to send to DeBot"
+                }
+            },
+            "unknown": {
+                "message": "Unknown error",
+                "hint": "Something went wrong. Please, send a bug report."
+            },
+            "interfaces": {
+                "notFound": {
+                    "message": "Interface with address \"{0}\" is not found",
+                    "hint": "Interface with ID \"{0}\" is not supported"
+                },
+                "failedToDecode": {
+                    "message": "Failed to decode DeBot body message",
+                    "hint": "Incorrect body in DeBot message"
+                },
+                "methodUnsupported": {
+                    "message": "Method \"{0}\" for interface \"{1}\" is unsupported",
+                    "hint": "Please, check the interface method name"
+                },
+                "failedToRunMethod": {
+                    "message": "Failed to run method \"{0}\"",
+                    "hint": "Please, check the interface method name"
+                },
+                "notSupported": {
+                    "message": "Interface is unsupported",
+                    "hint": "Interface with ID \"{0}\" is not supported"
+                },
+                "failedSendToDeBot": {
+                    "message": "Failed to send to DeBot",
+                    "hint": "Please, send a report about this problem"
+                },
+                "addressInput": {
+                    "failedToCreateMessage": {
+                        "message": "Failed to create Address Input Interactive message with no main account"
```

Comment:
```suggestion
                        "message": "Debot cannot create the address input interactive message because the main account isn't found"
```